src/index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/ecdbe10a83.js" crossorigin="anonymous"></script>
+   
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="container">
+           <!--  left container -->
+        <div class="left-side">
```

Comment:
Breaking down the code into the different sections makes it easy to follow along